### PR TITLE
docs: make editorial updates

### PR DESCRIPTION
**Pull request by SirSpidey**
Fixed some typos and grammar issues. Modified to fit better to IBM style.

### CODE REVIEW

**Review by finken2 — CHANGES_REQUESTED** [review]
@SirSpidey I realize this is ancient at this point, but if you're still interested in merging, would you mind updating the occurrences of satellite to Satellite?  It's a product name.

**Comment by SirSpidey** [comment]
@finken2 Thanks. Made that change and a few other minor editorial changes.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-- Create satellite location.
-- Create 6 VSIs with RHEL 7.9.
-- Assign the 3 hosts to the location control plane.
-- *Conditional creation*:
-  * Create a Red Hat OpenShift on IBM Cloud cluster and assign the 3 hosts to the cluster, so that you can run OpenShift workloads in your location.
-  * Configure a worker pool to an existing OpenShift Cluster.
+- Creates Satellite location.
+- Creates 6 VSIs with RHEL 7.9.
+- Assigns the three hosts to the location control plane.
+- *Conditionally creates* of these items:
```
Comment (finken2) [inline README.md:23]: grammar doesn't work here.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
-# IBM Cloud Satellite Module
+# IBM Cloud Satellite module
 
-Use this terrafrom automation to set up satellite location on IBM cloud.
-It will provision satellite location and create 6 VSIs and assign 3 host to control plane, and provision ROKS satellite cluster and auto assign 3 host to cluster,
-Configure cluster worker pool to an existing ROKS satellite cluster.
+Use this Terraform automation to set up a Satellite location on IBM cloud. The module provisions the IBM Cloud® Satellite location, creates 6 VSIs, assigns three hosts to the control plane, provisions a ROKS Satellite cluster, assigns three hosts to the cluster, and configures a cluster worker pool to an existing ROKS Satellite cluster.
```
Comment (finken2) [inline README.md:3]: some inconsistency on "IBM Cloud" here and a little lower down.  First sentence, Cloud should be capitalized.  Second sentence, the trademark symbol is there, but it's removed down on line 14.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
 
-This is a collection of sub modules that make it easier to provision a satellite on IBM Cloud.
-* location
-* host
-* cluster
-* configure-cluster-worker-pool
+This module is a collection of submodules that make it easier to provision a Satellite on IBM Cloud.
```
Comment (finken2) [inline README.md:5]: "provision a Satellite" doesn't really make sense.  We could say "provision Satellite resources" to cover the fact it does a location, hosts, cluster, etc

File: README.md [hunk 4]
```diff
@@ -33,22 +32,20 @@ https://cloud.ibm.com/docs/satellite?topic=satellite-getting-started
...
 This module is meant for use with Terraform 0.13 or later.
 
 ## Note
 
-* `location` module creates new location or use existing location ID/name. If user pass the location which is already exist,   satellite-location module will error out and exit the module. In such cases user has to set `is_location_exist` value to true. So that module will use existing location for processing.
-* All optional fields are given value `null` in varaible.tf file. User can configure the same by overwriting with appropriate values.
-* 'satellite-location' module download attach host script in the home directory and appends respective permissions to the script.
-* The modified script must be used in the `user_data` attribute of VSI instance
-* If we want to make use of a particular version of module, then set the argument "version" to respective module version.
+- The `location` submodule creates a location or uses an existing location ID or name. If you pass an existing location, the submodule produces an error and exits the module. To fix the issue, set `is_location_exist` to true so that the module uses the existing location.
```
Comment (finken2) [inline README.md:41]: i'm not sure why they originally wrote this from the standpoint of how to make it error. Up to you, but we could just state the two different possibilities: new location or use existing.

File: README.md [hunk 5]
```diff
@@ -33,22 +32,20 @@ https://cloud.ibm.com/docs/satellite?topic=satellite-getting-started
...
+- All optional fields are set to `null` in the `variable.tf` file. You can overwrite the values.
```
Comment (finken2) [inline README.md:42]: variables.tf (with an s).  Probably override, not overwrite.

File: README.md [hunk 6]
```diff
@@ -147,30 +144,38 @@ module "satellite-ibm" {
...
+- To destroy the VPC and all related resources, run the following command:
 
-`terraform destroy -var-file=./input.tfvars`
+  ````hcl
+  terraform destroy -var-file=./input.tfvars`
+  ```
 
-All optional parameters by default will be set to null in respective example's varaible.tf file. If user wants to configure any optional paramter he has overwrite the default value.
\ No newline at end of file
+All optional parameters are set to null by default in the example's `variable.tf` file. If you want to configure an optional parameter, overwrite the default value.
```
Comment (finken2) [inline README.md:181]: variables.tf w/ an s.
override instead of overwrite.

**Review by finken2 — CHANGES_REQUESTED** [review]

**Comment by SirSpidey** [comment]
@finken2 thanks for paying attention to this. I wasn't intending to rewrite this content because my team isn't responsible for this module. I was trying to suggest some small changes that I noticed while making some other changes through the "TIM" GH org.

**Review by finken2 — APPROVED** [review]

**Pull request merged**